scripts/bootstrap-plugin

```diff
@@ -21,8 +21,6 @@ if [ -d $pluginname ]; then
   err "There is already a plugin with that name. Cowardly refusing to overwrite it."
 fi
 
-buildConfigFilename=${pluginname}/build.config.js
-
 mvn archetype:generate -DarchetypeGroupId=org.graylog -DarchetypeArtifactId=graylog-plugin-archetype -DartifactId=$pluginname
 
-sed 's,\(../graylog2-server/graylog2-web-interface\),../\1/,' < ${buildConfigFilename}.sample > ${buildConfigFilename}
+echo ${pluginname}/ >> ${prefix}/.gitignore
```

Comment:
I am now keeping the modules in the dir, because of the issues with maven-archetype (https://issues.apache.org/jira/browse/ARCHETYPE-311) and management of pom modules, but I am now adding dirs to `.gitignore` only if they are not present yet.